Client/trunk/ParaEngineClient/3dengine/ParaEngineSettings.cpp

```diff
@@ -1264,6 +1264,29 @@ bool ParaEngine::ParaEngineSettings::Is64BitsSystem()
 	return sizeof(void*) > 4;
 }
 
+bool ParaEngine::ParaEngineSettings::Is64BitsProcess()
+{
```

Comment:
we do not need this function, there already is a function
![image](https://user-images.githubusercontent.com/94537/223691759-a8630240-f6d0-40e9-9e08-b7b6a77bfae8.png)

Reply:
just use sizeof(int) == 8